app/models/user.rb

```diff
@@ -5,6 +5,50 @@ class User < ApplicationRecord
   include Searchable
   include Storext.model
 
+  # NOTE: @citizen428 This is temporary code during profile migration and will
+  # be removed.
+  # rubocop:disable Metrics/BlockLength
+  concerning :ProfileMigration do
```

Comment:
`concerning` defines an anonymous concern and immediately includes it. This is my preferred way for concerns that aren't actually shared since it allows me to group related things without Rubocop complaining, but also doesn't introduce any indirection. And since most of this will be removed relatively soon, I'd rather give it more visibiltiy.

Reply:
TIL